.storybook/stories.test.js

```diff
@@ -0,0 +1,13 @@
+import initStoryshots from '@storybook/addon-storyshots';
```

Comment:
If you're interested, I did have a quick fifteen-minute attempt at linting for this with https://www.npmjs.com/package/eslint-plugin-header

Sadly our header looks too complex, and it's not particularly easy to test out this plugin! :stuck_out_tongue:    I tried using regex to have lines that match anything (for the lines that have the component name) but I didn't want to spend more time without  it being a ticket

Reply:
Yeah, it would be good to add something just to ensure the standard header is there. As for the dependency groups, there is are already rules for this in the WP ESLint plugin, but we disabled it a while back because it couldn't understand our aliases as being internal dependencies.